code/modules/mob/mob.dm

```diff
@@ -192,6 +192,25 @@
 	return 0
 
 
+/**
+ * Will handle a player cryoing or leaving the round in a similar way
+ *
+ */
+/mob/proc/handle_removal()
```

Comment:
Nitpick: This proc name is a bit generic, maybe something like `on_removal_from_round()`?